crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -6352,7 +6352,7 @@
         ]
       },
       "EventID": {
-        "description": "Unique ID of a Sui Event, the ID is a combination of tx seq number and event seq number, the ID is local to this particular fullnode and will be different from other fullnode.",
+        "description": "Unique ID of a Sui Event, the ID is a combination of tx seq number and event seq number.",
```

Comment:
change tx seq number to "transaction digest"

Reply:
Done